docs/podman-image-trust.1.md

```diff
@@ -14,18 +14,27 @@ podman\-trust - Manage container registry image trust policy
 REGISTRY[/REPOSITORY]
 
 # DESCRIPTION
-Manages which registries you trust as a source of container images based on its location.  The location is determined by the transport and the registry host of the image.  Using this container image `docker://docker.io/library/busybox` as an example, `docker` is the transport and `docker.io` is the registry host.
+Manages which registries you trust as a source of container images  based on its location.  The location is determined
+by the transport and the registry host of the image.  Using this container image `docker://docker.io/library/busybox`
+as an example, `docker` is the transport and `docker.io` is the registry host.
 
-The trust policy describes a registry scope (registry and/or repository).  This trust can use public keys for signed images.
+Trust is defined in **/etc/containers/policy.json** and is enforced when a user attempts to pull
+an image from a registry that is managed by policy.json.  The trust policy describes a registry scope (registry and/or repository).  This trust can use public keys for signed images.
```

Comment:
I’m not sure the “that is managed by …” clause makes sense — the code does not decide whether to read `policy.json` depending on contents of `policy.json`, that’s circular. It’s true, in a sense, because there is a mandatory “default” section, making every registry “managed” by the file in that weird sense, but I don’t think users should need to think about this.

If we need to qualify this at all, “when podman pulls an image, not when it is using an already locally-available one”, or maybe “when pulled by tools that support it, like Podman, Skopeo, Buildah, projectatomic/docker and others” would be conditions that users may really need to be aware of — but still, perhaps not worth bothering with here.

Reply:
OK, I think I'll change "attempts to pull an image" to "attempts to pull a remote image"

Reply:
I’m sorry, I was unclear — the part that reads as circular logic to me is “from a registry that is managed by policy.json”; _everything_ is managed by policy.json, IOW choosing whether to enforce policy.json based on contents of policy.json is already enforcing policy.json.

If you want to be more precise about trust being relevant only for pulls, or for remote pulls, that’s great, but I was trying to suggest dropping the ”… that is managed by policy.json” clause.

Reply:
OK, changed to:
```
Trust is defined in **/etc/containers/policy.json** and is enforced when a user attempts to pull
a remote image from a registry.  The trust policy in policy.json describes a registry scope (registry and/or repository) for the trust.  
```